packages/flutter/lib/src/services/text_formatter.dart

```diff
@@ -34,7 +70,11 @@ import 'text_input.dart';
 ///  * [FilteringTextInputFormatter], a provided formatter for filtering
 ///    characters.
 abstract class TextInputFormatter {
-  /// Called when text is being typed or cut/copy/pasted in the [EditableText].
+  /// Creates a new [TextInputFormatter].
+  const TextInputFormatter();
```

Comment:
Most of the input formatters in the framework can have const constructors. TODO: adding the const identifier once this goes in.